### DISCUSSION TITLE
Can we drag a column using column header instead of drag icon in Material-React-Table?

### DISCUSSION THREAD

**Discussion by Sandhu2995** [KevinVandy/material-react-table#1107] [Q&A]
### material-react-table version

2.12.1

### react & react-dom versions

18.0.0

### Describe the bug and the steps to reproduce it

I need to know whether we can drag and drop columns using column header.

I know, currently it can be achieved by using 'enableColumnOrdering' prop, but it brings the drag icon. I need to hide that drag icon, and still need the ability to drag columns.

### Minimal, Reproducible Example - (Optional, but Recommended)

This is from MRT documents - https://www.material-react-table.com/docs/guides/column-ordering-dnd

### Screenshots or Videos (Optional)

![image](https://github.com/KevinVandy/material-react-table/assets/166487908/331d0c88-7049-49b6-b702-798ca974645c)

### Do you intend to try to help solve this bug with your own PR?

None

### Terms

- [X] I understand that if my bug cannot be reliably reproduced in a debuggable environment, it will probably not be fixed and this issue may even be closed.

**Comment by KevinVandy** [comment]
You can just not turn on the built-in dragging features and then add your own drag events using `muiTableHeadCellProps`. It's a bit tedious though.
